packages/t/tomlplusplus/package.yml

```diff
@@ -0,0 +1,17 @@
+name       : tomlplusplus
+version    : 3.4.0
+release    : 1
+source     :
+    - https://github.com/marzer/tomlplusplus/archive/refs/tags/v3.4.0.tar.gz : 8517f65938a4faae9ccf8ebb36631a38c1cadfb5efa85d9a72e15b9e97d25155
+homepage   : https://marzer.github.io/tomlplusplus/
+license    : MIT
+component  : programming.library
+summary    : Header-only TOML config file parser and serializer for C++
```

Comment:
The description says "Header-only", but this is clearly building a shared library (`/usr/lib64/libtomlplusplus.so.3`). I'd pick a different summary/description if I were you.

Reply:
`TOML config file parser and serializer for C++` is fine. I checked the package website and apparently being header-only is an optional feature. Seems like nobody told whoever packaged it on Arch linux that since they build it as a shared library too :upside_down_face:

Reply:
This doesn't appear to be resolved.

Reply:
Still not resolved.